src/middleware/PromiseMiddleware.js

```diff
@@ -14,8 +9,8 @@ module.exports = function (
           .catch(this.req.next);
       };
 
-      this.app.response.renderAsync = function (view, properties = {}) {
-        return Promise.props(properties)
```

Comment:
Moved the "default" value of `{}` to a "fallback" value because the default value only gets used when the argument is `undefined` or not provided, but if `null` is provided, it will fail.